.github/workflows/tests.yaml

```diff
@@ -75,7 +75,102 @@ jobs:
           # Make sure to add to GitHub secrets!
           token: ${{secrets.CODECOV_TOKEN}}
           file: ./directory/coverage.txt
+ 
+  gds_admin_test:
+    name: GDS Admin UI Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.8.0
+          cache: "yarn"
+          cache-dependency-path: ./web/gds-admin-ui/yarn.lock
+
+      - name: Install Dependencies
+        working-directory: ./web/gds-admin-ui
+        run: yarn 
+
+      - name: Run Tests
+        working-directory: ./web/gds-admin-ui
+        env: 
+          #maybe we can make it dynamic like ${{secrets.REACT_APP_GDS_API_ENDPOINT}}
+          REACT_APP_GDS_API_ENDPOINT : http://localhost:4434/v2 
+          REACT_APP_GDS_IS_TESTNET : true
+        run: yarn test
+
+      - name: Upload Coverage report to CodeCov
+        uses: codecov/codecov-action@v1.0.0
+        with:
+          # Make sure to add to GitHub secrets!
+          token: ${{secrets.CODECOV_TOKEN}}
+  
+  gds_ui_test:
+    name: GDS UI Tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.8.0
+          cache: "yarn"
+          cache-dependency-path: ./web/gds-ui/yarn.lock
+
+      - name: Install Dependencies
+        working-directory: ./web/gds-ui
+        run: yarn 
+
+      - name: Run Tests
+        working-directory: ./web/gds-ui
+        env: 
+          #maybe we can make it dynamic like ${{secrets.REACT_APP_GDS_API_ENDPOINT}}
+          REACT_APP_GDS_API_ENDPOINT : http://localhost:8080 
+          REACT_APP_GDS_IS_TESTNET : true
+        run: yarn test
+
+      - name: Upload Coverage report to CodeCov
+        uses: codecov/codecov-action@v1.0.0
+        with:
+          # Make sure to add to GitHub secrets!
+          token: ${{secrets.CODECOV_TOKEN}}
+
+  lingui_gds_ui:
```

Comment:
So the point of this test is to ensure the messages for i18n are correct and that lingui builds correctly right? What would you think about adding rumi to this so that we have a report of our translation coverage? 

https://github.com/rotationalio/rumi